templates/zerver/help/import-from-slack.md

```diff
@@ -0,0 +1,50 @@
+# Import from Slack
+
+To import users and channels from Slack to Zulip please follow
+bellow steps:
+
+1. Generate Slack API Token from [here](https://api.slack.com/docs/oauth-test-tokens).
+   Test tokens are enough to import all of needed data.
+
+2. Go to **Settings** by clicking on cog from top right corner.
+
+    ![Settings](/static/images/help/cog_dropdown_settings.png)
+
+3. Scroll down to **Your bots** section.
+
+    ![Your bots](/static/images/help/settings-your-bots.png)
+
+4. Click on **Show/change your API key** button, provide your password
+   and copy generated API key.
+
```

Comment:
Make another step along the lines of "Upon clicking the `**Show/change your API key**` button, you will be asked to confirm your identity by entering your password in the`**Current password**` field."

After that, make another step about clicking the `**Get API Key**` to get your API key once you enter your password.